docs/preview/01-index.md

```diff
@@ -6,17 +6,18 @@ sidebar_label: Welcome
 ---
 
 # Introduction
-Arcus Messaging is a library that helps with the integration of messaging systems like queues or event subscriptions and allows you to focus on implementing business logic when processing messages instead of spending time with message peaking, connections, deserialization, and other infrastructure code that takes up time.
+Welcome to the Arcus Messaging site! 🎉
 
-## Guides
-* Migrate from v0.x to v1.0 ([docs](./03-Guides/migration-guide-v1.0.md))
+# What is Arcus Messaging?
+Arcus Messaging is an umbrella term for a set of NuGet packages `Arcus.Messaging.*` that helps with abstracting away boilerplate code when integrating with messaging systems like Azure Service bus. It allows you to focus on implementing business logic instead of spending ime with message picking, connections, deserialization, and the like.
```

Comment:
```suggestion
Arcus Messaging is an umbrella term for a set of NuGet packages `Arcus.Messaging.*` that helps with abstracting away boilerplate code when integrating with messaging systems like Azure Service Bus. It allows you to focus on implementing business logic instead of spending time with message picking, connections, deserialization, and the like.
```